src/client/jest.config.js

```diff
@@ -8,6 +8,22 @@ module.exports = {
   globals: {
     "ts-jest": {
       babelConfig: true,
+      diagnostics: {
+        ignoreCodes: [1343], // otherwise throws error about "import.meta" before babel can help
+      },
     },
   },
+  // for jest > 27 ..
```

Comment:
if switching to vitest works out, we can get rid of (all) this .. otherwise it is handy to know what to tweak for later Jests ..